uint64 rev_number

EntityUpdateInfo[] update_entities
string[]    remove_entities
